imports/plugins/included/payments-example/client/settings/containers/exampleSettingsFormContainer.js

```diff
@@ -0,0 +1,90 @@
+import { composeWithTracker } from "/lib/api/compose";
```

Comment:
Import order is npm first, Meteor imports next, then local project imports last. So basically just need to move the React import first, then Meteor. Also no blank lines in between.